dataactcore/models/stagingModels.py

```diff
@@ -1,15 +1,261 @@
+    transactionobligatedamount = Column(Numeric)
+    uri = Column(Text)
+    ussgl480100_undeliveredordersobligationsunpaid_cpe = Column(Numeric)
+    ussgl480100_undeliveredordersobligationsunpaid_fyb = Column(Numeric)
+    ussgl480200_undeliveredordersobligationsprepaidadv_cpe = Column(
+        Numeric, key="ussgl480200_undeliveredordersobligationsprepaidadvanced_cpe")
+    ussgl480200_undeliveredordersobligationsprepaidadvanced_fyb = Column(Numeric)
+    ussgl483100_undeliveredordersobligtransferredunpaid_cpe = Column(
+        Numeric, key="ussgl483100_undeliveredordersobligationstransferredunpaid_cpe")
+    ussgl483200_undeliveredordersobligtransferredppdadv_cpe = Column(
+        Numeric, key="ussgl483200_undeliveredordersobligationstransferredprepaidadvanced_cpe")
+    ussgl487100_downadjsprioryrunpaidundelivordersobligrec_cpe = Column(
+        Numeric, key="ussgl487100_downwardadjustmentsofprioryearunpaidundeliveredordersobligationsrecoveries_cpe")
+    ussgl487200_downadjsprioryrppdadvundelivordersobligref_cpe = Column(
+        Numeric, key="ussgl487200_downwardadjustmentsofprioryearprepaidadvancedundeliveredordersobligationsrefundscollected_cpe")
+    ussgl488100_upadjsprioryearundelivordersobligunpaid_cpe = Column(
+        Numeric, key="ussgl488100_upwardadjustmentsofprioryearundeliveredordersobligationsunpaid_cpe")
+    ussgl488200_upadjsprioryrundelivordersobligprepaidadv_cpe = Column(
+        Numeric, key="ussgl488200_upwardadjustmentsofprioryearundeliveredordersobligationsprepaidadvanced_cpe")
+    ussgl490100_deliveredordersobligationsunpaid_cpe = Column(Numeric)
+    ussgl490100_deliveredordersobligationsunpaid_fyb = Column(Numeric)
+    ussgl490200_deliveredordersobligationspaid_cpe = Column(Numeric)
+    ussgl490800_authorityoutlayednotyetdisbursed_cpe = Column(Numeric)
+    ussgl490800_authorityoutlayednotyetdisbursed_fyb = Column(Numeric)
+    ussgl493100_deliveredordersobligstransferredunpaid_cpe = Column(
+        Numeric, key="ussgl493100_deliveredordersobligationstransferredunpaid_cpe")
+    ussgl497100_downadjsprioryrunpaiddelivordersobligrec_cpe = Column(
+        Numeric, key="ussgl497100_downwardadjustmentsofprioryearunpaiddeliveredordersobligationsrecoveries_cpe")
+    ussgl497200_downadjsprioryrpaiddelivordersobligrefclt_cpe = Column(
+        Numeric, key="ussgl497200_downwardadjustmentsofprioryearpaiddeliveredordersobligationsrefundscollected_cpe")
+    ussgl498100_upadjsprioryeardeliveredordersobligunpaid_cpe = Column(
+        Numeric, key="USSGL498100_UpwardAdjustmentsOfPriorYearDeliveredOrdersObligationsUnpaid_CPE")
+    ussgl498200_upadjsprioryrdelivordersobligpaid_cpe = Column(
+        Numeric, key="ussgl498200_upwardadjustmentsofprioryeardeliveredordersobligationspaid_cpe")
+    tas = Column(Text, nullable=False, default=concatTas, onupdate=concatTas)
+    Index("ix_award_financial_tas_oc_pa", "tas", "objectclass", "programactivitycode", unique=False)
```

Comment:
Did a bit of research, this seems fine now for TAS, though this will not help if we have cases where we are querying for object class or program activity alone, or a case where we query by TAS + PA without OC.  I'll look at the few cross-file rules we have right now and see if we're breaking this pattern yet.